llpc/CMakeLists.txt

```diff
@@ -391,6 +393,13 @@ add_executable(amdllpc tool/amdllpc.cpp)
 add_dependencies(amdllpc llpc_standalone_compiler)
 target_link_libraries(amdllpc PRIVATE llpc_standalone_compiler)
 set_compiler_options(amdllpc ${LLPC_ENABLE_WERROR})
+#if USING_STATIC_SPVGEN
```

Comment:
nit: remove the comments?

Reply:
> Also, is there a plan to eventually deprecate using spvgen as a shared library and remove the related code from amdllpc?

If there is nothing wrong with the static library, I will add other changes to deprecate shared library. 

> How do you plan to handle the tests in `llpc/llpc/test/shaderdb/error_reporting` that check that we print an error when a bad spvgen direcotry is provided or spvgen is not specified at all? Do we need to introduce a new `requires` lit attribute?

We may remove this directory.

Reply:
If we want both the dynamic/shared and static compilation options to co-exist for some time, then I don't think we should just delete these test. Refactoring related to the static library may introduce unexpected breakage in the shared library.